src/bugsnag/BugsnagErrorBoundary.tsx

```diff
@@ -0,0 +1,38 @@
+// NOTE: do not include any other files here, try to keep this component separate from the rest
+
+import React from "react";
+import { OnErrorCallback } from "@bugsnag/core";
+import Bugsnag from "@bugsnag/js";
+
+import { isBugsnagStarted } from "./init";
+import { ErrorPrompt } from "./ErrorPrompt";
+
+interface BugsnagErrorBoundaryProps {
+  onError?: OnErrorCallback;
+  FallbackComponent?: React.ComponentType<{
+    error: Error;
+    info: React.ErrorInfo;
+    clearError: () => void;
+  }>;
+}
+
+const ErrorBoundaryStub: React.FC<BugsnagErrorBoundaryProps> = ({
+  children,
+}) => {
+  // NOTE: has same interface as Bugsnag's error boundary component to prevent React complaining props are being set to a React.Fragment
+  return <>{children}</>;
+};
+
+const MaybeErrorBoundary = isBugsnagStarted
```

Comment:
Does it need to be started before calling this?

Reply:
Helps avoiding console messages of the kind:
```
Bugsnag.addOnError() was called before Bugsnag.start()
Bugsnag.getPlugin() was called before Bugsnag.start()
```

Reply:
Though this will probably also create a race condition, where if Bugsnag isn't started at the point this is called, then it will never be used. Which I expect is not the desired intent here.

Reply:
The check is still the original one, and hasn't been done based on Bugsnag itself being started or not, only if an API key is provided (even a wrong one).

 The only outcome of a race condition would be to yet again see warning such
`Bugsnag.addOnError() was called before Bugsnag.start()`